lib/widgets/login.dart

```diff
@@ -373,8 +373,15 @@ class _PasswordLoginPageState extends State<PasswordLoginPage> {
       decoration: InputDecoration(
         labelText: 'Password',
         helperText: kLayoutPinningHelperText,
-        suffixIcon: Semantics(label: 'Hide password', toggled: _obscurePassword,
+        // TODO(material-3): Simplify away `Semantics` by using IconButton's
+        //   M3-only params `isSelected` / `selectedIcon`, after fixing
+        //   https://github.com/flutter/flutter/issues/127145 . (Also, the
```

Comment:
Cool, good to have reported this. Looks like that issue was assigned just today, so hopefully will get fixed soon.